### PR TITLE
Bugfix/webpublisher task type

### PR DESCRIPTION
**Pull request by kalisp**
Capitalize task type 
Introduce default frameStart, frameEnd for single frame
Temporary files in temp folders

### CODE REVIEW

File: openpype/hosts/webpublisher/plugins/publish/collect_published_files.py [hunk 1]
```diff
@@ -174,6 +177,8 @@ def _get_family(self, settings, task_type, is_sequence, extension):
                 (family, [families], subset_template_name, tags) tuple
                 AssertionError if not matching family found
         """
+        if task_type:
```
Comment (iLLiCiTiT) [inline openpype/hosts/webpublisher/plugins/publish/collect_published_files.py:180]: Wouldn't be better to lower all keys in `settings` and `task_type` instead if expecting that only one character is wrong?

**Review by 64qam — APPROVED** [review]

File: openpype/hosts/webpublisher/plugins/publish/collect_published_files.py (hunk 1 above)
Comment (jakubjezek001) [reply openpype/hosts/webpublisher/plugins/publish/collect_published_files.py:180]: I agree, this would be much safer to work with

Comment (mkolar) [reply openpype/hosts/webpublisher/plugins/publish/collect_published_files.py:180]: agreed as well

**Review by mkolar — CHANGES_REQUESTED** [review]
let's change the task_type to lowercase as suggested please

**Review by iLLiCiTiT — APPROVED** [review]
Looks ok

**Review by mkolar — APPROVED** [review]

**Pull request merged**